templates/zerver/help/status-and-availability.md

```diff
@@ -65,26 +66,37 @@ There are four possible availabilities:
 * **Idle** (<span class="indicator orange"></span>): Zulip is open on
   your computer (either desktop or web), but you are not active.
 
-* **Offline** (<span class="indicator grey"></span>): Zulip is not
-  open on your computer.
+* **Offline** (<span class="indicator grey"></span>): Zulip is not open
+  on your computer, or you have turned on invisible mode.
 
-* **Unavailable** (<span class="indicator grey-line"></span>): You can
-  always manually set your availability to unavailable.
+You can see when someone offline was last recorded as active by hovering
+over their name in the left or right sidebar.
 
+### Invisible mode
 
-You can see when someone was last recorded as active by hovering over
-their name in the left or right sidebar (even if the user is marked as
-unavailable).
+Zulip supports the privacy option of never updating the availability
+information for your account. The result is that you will always
+appear to other users as **Offline**, regardless of your activity in
+Zulip.
+
+With this setting, your "Last active" time displayed to other users in
+the UI will be frozen as the approximate time you enabled this setting.
+
+!!! warn ""
+
+    **Note**: Because this setting works by making your availability
+    stop updating, you'll still appear to other users as active for a
+    few minutes after turning on invisible mode.
```

Comment:
I think this note can be deleted -- we now sync an event to all users immediately, since 637867dad131d15fda1cf8d4ca132adf3b8444bb

Reply:
I thought this was referring to the slight delay on the frontend (which is 2 minutes max), but I can definitely remove it.

Reply:
Right, so just to be clear, what that PR does is sends an update to your availability saying "I was last idle OFFLINE_THRESHOLD_SECS + 2 minutes ago" -- which is long enough that the frontend should display the user as offline immediately. If you've tested and that's not accurate, we have a bug to track down :).

Reply:
Yeah, the frontend code of `presence.js` doesn't even look at the idle timestamp if the active timestamp was less than the threshold for offline.

I'm AFK at the moment, but I think it's where we get status from raw. Or something like that. 

So, we need to fix that logic.